src/zookeeper/config/zookeeper.py

```diff
@@ -48,13 +47,15 @@ def run(self):
         zookeeper_com = dict()
         zookeeper_com["host-list"] = list()
         zookeeper_com["quorum"] = ""
+        zookeeper_com["zk-servers"] = ""
 
         for host_config in self.cluster_configuration["machine-list"]:
             if "pai-master" in host_config and host_config["pai-master"] == "true":
                 zookeeper_com["host-list"].append(host_config["hostname"])
                 if zookeeper_com["quorum"] != "":
                     zookeeper_com["quorum"] = zookeeper_com["quorum"] + ","
                 zookeeper_com["quorum"] = zookeeper_com["quorum"] + host_config["hostip"] + ":2181"
+                zookeeper_com["zk-servers"] = zookeeper_com["zk-servers"] + host_config["hostip"] + ":2888:3888\n"
```

Comment:
This new value should be added into the document in this path about zk config. https://github.com/Microsoft/pai/blob/master/src/zookeeper/config/zookeeper.md

Reply:
Got it, I will update.